Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,27 @@
+class Gcviewer < Formula
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/zipball/master"
+  version "1.34.1"
+  sha256 "775430498a8a38d6bce59e265b948ff47fafb5e00aea3fc5d3c47adcd650a36c"
+
+  depends_on "maven" => :build
+
+  def install
+    libexec.install Dir['*']
+    cd libexec
+
+    system "mvn clean install"
+
+    (bin / "gcviewer.sh").write <<-EOS.undent
```

Comment:
Also, this'll be done for you automatically if you just use `bin.write_jar_script libexec/"target/gcviewer-#{version}.jar", "gcviewer"`.